javascript-intro-challenges-stevie-dane.js

```diff
@@ -0,0 +1,39 @@
+//1
+console.log(34+71)
+
+//2
+console.log(123-67)
+
+//3
+console.log(56*23)
+
+//4
+console.log(45/5)
+
+//5
+console.log(Math.pow(5,7))
+
+//6
+console.log(33%6)
+
+//7
+console.log("stevie".length)
+
+//8
+console.log("stevie".includes("e"))
+
+//9
+console.log()
+
+//10
+console.log
+
+//11
+console.log
+
+//12
+console.log(number > null || number ≤ null) 
+
```

Comment:
Review boolean values. Helpful hint. Copy/paste the challenge questions to maintain your place as well as for reviewing your notes later